module2/projects/little_shop/index.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: page
+title: Little Shop
+length: 1.5 weeks
+tags:
+type: project
+---
+
+## Project Description
+
+You are working for a company developing an E-Commerce Application. Your team is working in a service-oriented architecture, meaning the front and back ends of this application are separate and communicate via APIs. 
+
+Your job is to expose the data that powers the site through an API that the front end would consume. You won't be building an actual front end for this project, so you can look at [these Wireframes](./wireframes) to get an idea of how the front end would integrate with your back end.
+
+<!-- TO-DO: Add details about connecting to an already existing FE app that will consume these endpoints -->
+
+## Learning Goals
+
+### Explicit
+
+* Use ActiveRecord and SQL to write queries that deal with one-to-many database relationships
+* Expose API endpoints to CRUD database resources
+* Validate models and handle sad paths for invalid data input
+* Test both happy and sad path functionality based on JSON contracts
+* Use MVC to organize code effectively, lmiting data logic in controllers and serializers
+* Track user stories with GitHub Projects
+* Deploy your application to the internet (?)
+* FE LEARNING GOALS
+
+### Implicit
+Below are skills that are more general/necessary on the job that are practiced by completing the goals above.
+
+* Reading Documentation
+* Time Management
+* Prioritizing Work
+* Breaking down large project into small pieces
+* Breaking down a problem into small steps
+* Practice individual research (articles, videos, mentors)
+
+## Project Management
+
+Groups must utilize a project board in order to organize areas of work and allow for  visibility of each other's progress. In addition, groups will be evaluated on the evidence of meaningful code review, either via comments on a PR, or through discussion of live code review during their evaluation. 
+
+## Postman
+
+We have adapted this project from previous cohorts and have rewritten a fully functional test suite right within Postman. By importing the JSON files noted below, you can open the Postman "Runner" and execute a large suite of tests.
```

Comment:
Is it important to note that we've adapted this from previous cohorts and rewritten the test suite?  Rather than just stating the second sentence?

Reply:
Funnily enough that's actually carry over from who knows when. But I agree with your sentiment. I'm not sure there's value in keeping that line.

Reply:
Yeah great point -- not necessary at all! Just a copy/paste from old project and happy to take it out